src/page/sinfo.rs

```diff
@@ -32,28 +32,39 @@ impl core::fmt::Display for SecInfo {
         match self.class {
             Class::Secs => write!(f, "S"),
             Class::Tcs => write!(f, "T"),
-            Class::Reg => write!(f, "{}", self.flags),
-            Class::Va => write!(f, "V"),
-            Class::Trim => write!(f, "^"),
+            Class::Regular => write!(f, "{}", self.flags),
+            Class::VersionArray => write!(f, "V"),
+            Class::Trimmed => write!(f, "^"),
+            Class::ShadowStackFirst => write!(f, "F"),
+            Class::ShadowStackRest => write!(f, "R"),
```

Comment:
What if we used two letters, given that there three page types with the letter S: `SE`, `TC`, `RE`, `TR`, `SF` and `SR`?

Reply:
I would prefer just one. The goal is to allows the shortest possible way to capture the class/flags when debugging output. You're really going to want to dump things like addresses along with these letters in a single line. With just two addresses in hex, a class and flags (plus spaces to delimit fields) you've already consumed half the width of a terminal.

Reply:
Can you propose letters for shadow stack entries?

Reply:
I'll go with those.